Makefile

```diff
@@ -126,6 +147,29 @@ push: push-ccm-image push-node-image
 .PHONY: push-images
 push-images: push-ccm-image push-node-image
 
+.PHONY: push-node-manager-manifest
+push-node-manager-manifest:
+	docker manifest create --amend $(NODE_MANAGER_IMAGE) $(ALL_NODE_MANAGER_IMAGES)
+	docker manifest annotate --os linux --arch amd64 $(NODE_MANAGER_IMAGE) $(NODE_MANAGER_LINUX_IMAGE)
+	# For Windows images, we also need to include the "os.version" in the manifest list, so the Windows node can pull the proper image it needs.
+	# At the moment, docker manifest annotate doesn't allow us to set the os.version, so we'll have to it ourselves. The manifest list can be found locally as JSONs.
+	# See: https://github.com/moby/moby/issues/41417
```

Comment:
Do we still need this given https://github.com/docker/cli/pull/2578 has merged?

Reply:
seems it's in `20.10.0`, that could make the logic t quite simple

Reply:
Right, I've updated this, so that we'll be using ``docker manifest annotate --os-version`` instead.

Reply:
Thank you @claudiubelu. LGTM from me.